app/components/windows/AdvancedStatistics.tsx

```diff
@@ -0,0 +1,194 @@
+import { Component } from 'vue-property-decorator';
+import { Inject } from 'services/core/injector';
+import TsxComponent from 'components/tsx-component';
+import { $t } from 'services/i18n';
+import cx from 'classnames';
+import styles from './AdvancedStatistics.m.less';
+import ModalLayout from 'components/ModalLayout.vue';
+import PerformanceMetrics from './../PerformanceMetrics.vue';
+import { StreamingService, EStreamingState } from 'services/streaming';
+import GlobalSyncStatus from 'components/GlobalSyncStatus.vue';
+import moment from 'moment';
+import { Subscription } from 'rxjs';
+import { PerformanceService, EStreamQuality } from 'services/performance';
+import {
+  ENotificationType,
+  ENotificationSubType,
+  NotificationsService,
+  INotification,
+} from 'services/notifications';
+
+@Component({})
+export default class AdvancedStatistics extends TsxComponent<{}> {
+  @Inject() notificationsService: NotificationsService;
+  @Inject() performanceService: PerformanceService;
+  @Inject() streamingService: StreamingService;
+
+  private updateInterval = 0;
+  private notifications: INotification[] = [];
+  private streamingStatus: EStreamingState = null;
+  private notificationPushed: Subscription = null;
+  private streamingStatusChange: Subscription = null;
+
+  mounted() {
+    this.notificationPushed = this.notificationsService.notificationPushed.subscribe(notify => {
+      this.onNotificationHandler(notify);
+    });
+
+    this.streamingStatusChange = this.streamingService.streamingStatusChange.subscribe(status => {
+      this.onStreamingStatusChange(status);
+    });
+
+    this.notifications = this.notificationsService
+      .getAll()
+      .filter(notification => notification.subType !== ENotificationSubType.DEFAULT);
+    // update the time labels
+    this.updateInterval = window.setInterval(() => {
+      this.$forceUpdate();
+    }, 60 * 1000);
+  }
+
+  destroyed() {
+    this.notificationsService.markAllAsRead();
+    this.notificationPushed.unsubscribe();
+    this.streamingStatusChange.unsubscribe();
+    clearInterval(this.updateInterval);
+  }
+
+  get streamQuality() {
+    return this.performanceService.streamQuality;
+  }
+
+  get status(): { type: string; description: string } {
+    if (this.streamingStatus === 'reconnecting' || this.streamQuality === EStreamQuality.POOR) {
```

Comment:
should be `this.streamingStatus === EStreamingState.Reconnecting` right?

Reply:
Yep, will fix